### DISCUSSION TITLE
Paying off invoice not an option

### DISCUSSION THREAD

**Discussion by MenooMeow** [frappe/books#728] [General]
I was following the guide on the website regarding creating invoices and then paying the invoice.  When I visit Sales --> Sales Invoices --> click on an invoice --> 3 dots -->  I only get the option to "Cancel".  I do not see "Make Payment" like it is explained in "Making Sales Payments" in the instructions:  https://docs.frappebooks.com/transactions/sales-invoices.html

Did I miss a configuration step?

**Comment by 18alantom** [comment]
The docs are slightly outdated. You'll have to click on the Create button to make a linked payment.
![Screenshot 2023-09-12 at 15 28 07](https://github.com/frappe/books/assets/29507195/7aeaa1f7-14e2-4812-ac88-3c7851b6ab8a)

**Reply by MenooMeow** [reply]
Thanks, that worked perfectly.
